README.md

```diff
@@ -31,11 +38,23 @@ test.put('hello', 'world', function() {
 
 ## API
 
-#### `subdb = sub(db, [prefix], [options])`
+### `subdb = sub(db, [prefix], [opts])`
+
+Returns a `levelup` instance that uses subleveldown with `prefix` on top of the underlying *down of `db`. Each sub level is a `levelup` of its own which can have specific encodings.
```

Comment:
Maybe

```patch
- underlying *down of `db`
+ underlying store
```

Reply:
Or "that wraps the underlying store with `subleveldown` to prefix its keys with `prefix`".